### PR TITLE
Suppress wp-pointer using css

### PR DESCRIPTION
**Pull request by mamatharao05**
Added hook to suppress WP-Form notice using CSS for Bluehost, Hostgator and Crazy domains.

Ticket: https://jira.newfold.com/browse/PRESS0-1248

Before
image

After
image

### CODE REVIEW

File: includes/ECommerce.php [hunk 1]
```diff
@@ -675,4 +675,19 @@ function check_url_match( $brand_name, $site_url ) {
...
+
+	/**
+	 * Suppress WP-Form Notice using css
+	 *
+	 * @return void
+	 */
+
+	public function hide_wp_pointer_with_css() {
+		$brand = $this->container->plugin()->id;
+		if ( 'bluehost' === $brand || 'hostgator' === $brand ) {
```
Comment (aratidgr8) [inline includes/ECommerce.php:687]: Are we using `Yoda` in Ecommerce as well?

Comment (ramyakrishnai) [inline includes/ECommerce.php:687]: Don't we need this for `crazy-domains`

Comment (mamatharao05) [reply includes/ECommerce.php:687]: @aratidgr8 This is the newfold standard. So I followed here too.

Comment (mamatharao05) [reply includes/ECommerce.php]: Updated. Applicable for Bluehost, Hostgator and crazy domains.

File: includes/ECommerce.php [hunk 2]
```diff
@@ -106,8 +106,8 @@ public function __construct( Container $container ) {
 		add_filter( 'manage_edit-page_sortable_columns', array( $this, 'sortable_columns' ) );
 		add_action( 'wp_login', array( $this, 'show_store_setup' ) );
 		add_action( 'auth_cookie_expired', array( $this, 'show_store_setup' ) );
-
-		add_action('admin_enqueue_scripts', array( $this, 'set_wpnav_collapse_setting'));			
+		add_action('admin_head', array( $this, 'hide_wp_pointer_with_css' ) );
+		add_action('admin_enqueue_scripts', array( $this, 'set_wpnav_collapse_setting'));
 
 		if ( ( $container->plugin()->id === 'bluehost' && ( $canAccessGlobalCTB || $hasYithExtended ) ) || ( $container->plugin()->id === 'hostgator' && $hasYithExtended ) ) {
```
Comment (aratidgr8) [inline includes/ECommerce.php:112]: No Yoda here?

Comment (mamatharao05) [reply includes/ECommerce.php:112]: I haven't made any changes to the existing code.

**Review by aratidgr8 — APPROVED** [review]

**Pull request merged**